### DISCUSSION TITLE
[Idea] Ordered Lists on Selected Text

### DISCUSSION THREAD

**Discussion by fuzenco** [zed-industries/zed#7002] [Feature Requests]
I find myself copying text into Zed that should be turned into Ordered Lists—not for html but for placing into text docs. I’d love to see a command that prepends 1., 2., 3., etc. or A., B., C. etc. to the beginning of lines that I run the Split Text Into Lines command.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
